.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: crystal
```

Comment:
I decided to remove it since the crystal v0.23.1 works on Travis. Redis server was not being installed with the docker option.